docs/usage.md

```diff
@@ -0,0 +1,89 @@
+# Arcadia-Science/hifi2genome Usage
+
+This page provides documentation on how to use the Arcadia-Science/hifi2genome workflow.
+
+## Input specifications
+
+This pipeline only process PacBio Hifi sequences in a single file in fastq format. The hifi2genome workflow takes a CSV samplesheet listing the sample names and paths to the fastq files as direct input. It does not take the path to the directory of the fastq files, you must list them in a samplesheet. The CSV must contain the columns:
+
+`sample, fastq`
+Neither the `sample` or `fastq` columns may contain spaces, and the `fastq` column contains the path to your PacBio Hifi reads in a single fastq file. The path to the fastq file can be the full path to a local file or s3 URI. Valid examples look like the following:
+
+```
+sample,fastq
+acineobacter_baumanii_AYE,s3://nf-test-datasets/hifi2genome/raw_reads/m64004_210929_143746.bc2001.fq.gz
+bacillus_cereus_971,s3://nf-test-datasets/hifi2genome/raw_reads/m64004_210929_143746.bc2002.fq.gz
+bacillus_subtilis_w23,s3://nf-test-datasets/hifi2genome/raw_reads/m64004_210929_143746.bc2003.fq.gz
+burkholderia_cepacia_ucb717,s3://nf-test-datasets/hifi2genome/raw_reads/m64004_210929_143746.bc2004.fq.gz
+```
+
+The samplesheet must be a valid `.csv` file extension with 2 comma-separated columns names as above, and the sample IDs must be unique. The pipeline requires fastq files to end in `.fastq` or `.fq` or compressed `.fastq.gz` or `.fq.gz`.
+
+## Running the pipeline
+
+A typical command for running the pipeline looks like:
+`nextflow run Arcadia-Science/hifi2genome --input samplesheet.csv --outdir <OUTDIR> -profile docker`
+
+This will launch the pipeline with the `docker` configuration pipeline. See below for more information about profiles.
+The pipeline will create the following files and directories in your working directory:
+
+```
+work                # Directory containing the nextflow working files
+<OUTIDR>            # Finished results in specified location (defined with --outdir)
+.nextflow_log       # Log file from Nextflow
+# Other nextflow hidden files, eg. history of pipeline runs and old logs.
+```
+
+### Updating the pipeline
+
+When you run the above command, Nextflow automatically pulls the pipeline code from Github and stores it as a cached version. When running the pipeline after this, it will always use the cached version if available - even if the pipeline has been updated since. To make sure you are running the latest version of the pipeline, regularly update the cached version with:
```

Comment:
```suggestion
When you run the above command, Nextflow automatically pulls the pipeline code from GitHub and stores it as a cached version. When running the pipeline after this, it will always use the cached version if available - even if the pipeline has been updated since. To make sure you are running the latest version of the pipeline, regularly update the cached version with:
```

Reply:
Maybe instead of cached version, use "local version"?